bin/pasta_show_maintainers.py

```diff
@@ -0,0 +1,286 @@
+                                          '(default: %(default)s)')
+    args = parser.parse_args()
+    d_repo = config.repo_location
+    repo = pygit2.Repository(d_repo)
+    commit_version = args.commit if args.commit else 'HEAD'
+    commit = repo.revparse_single(commit_version)
+    tree = commit.tree
+        
+    if args.filter:
+        filter_by_files = file_to_string(args.filter).splitlines()
+        # Automatically find kernel version from file name "v5.6"
+        version_regex = re.compile(r'v[0-9]+.[0-9]+')
+        kernel_version = version_regex.findall(args.filter)[0]
+
+    if args.file:
+        all_maintainers = file_to_string(args.file)
+        LinuxMaintainers(all_maintainers)
+    else:
+        if kernel_version:
+            all_maintainers = load_maintainers(
+                config, [kernel_version]).get(kernel_version)
+        else:
+            all_maintainers = tree['MAINTAINERS'].data.decode('iso8859')
+
+    all_filenames = walk_commit_tree(tree)
+
+    global _tmp_tree
+    _tmp_tree = tree
+
+    processes = int(cpu_count() * args.cpu_factor)
+    p = Pool(processes=processes, maxtasksperchild=1)
+    result = p.map(get_filesize, all_filenames)
+    p.close()
+    p.join()
+   
+    file_sizes = dict(result)
+    
+    _tmp_tree = None
+
+    if args.entries:
+        title = ['Entries']
+        if args.filter != None:
+            counts, irrelevant_dirs, relevant_dirs = get_counts_by_entry(
+                all_maintainers, file_sizes, filter_by_files)
+        else:
+            counts, _, _= get_counts_by_entry(all_maintainers, file_sizes)
+    else:
+        title = ['Maintainers']
+        if args.filter != None:
+            counts, maintainer_to_subsystem = get_counts_by_maintainer(
+                all_maintainers, file_sizes, args.verbose, filter_by_files)
+        else:
+            counts, maintainer_to_subsystem = get_counts_by_maintainer(
+                all_maintainers, file_sizes, args.verbose)
+
+    fields = dict()
+    fields['Maintainers'] = fields['Entries'] = lambda item : item[0]
+    fields['LoC in list'] = lambda item : item[1].loc_filt
+    fields['Total LoC'] = fields['LoC'] = lambda item : item[1].loc
+    fields['Byte count'] = lambda item : item[1].byte
+    fields['In list(%)']=lambda item:round((item[1].loc_filt/item[1].loc)*100,2)
+    fields['Status'] = lambda item : status(all_maintainers, item[0])
+    fields['Entries of maintainer']=lambda item:', '.join(maintainer_to_subsystem[item[0]])
+    fields['Irrelevant directories'] = lambda item: ', '.join(irrelevant_dirs[item[0]])
+    fields['Relevant directories'] = lambda item: ', '.join(relevant_dirs[item[0]])
+   
+    title.append('LoC')
+    if not args.smallstat:
+        if args.filter != None:
+            title += ['LoC in list', 'In list(%)']
+            if args.entries:
+                title += ['Irrelevant directories', 'Relevant directories']
+        if args.bytes:
```

Comment:
If you move this a bit above, you can simply use append or title += ['Byte count'].